.rubocop.yml

```diff
@@ -82,7 +82,7 @@ CollectionMethods:
 
 # Offence count: 23
 CyclomaticComplexity:
-  Max: 31
+  Max: 40
```

Comment:
I was told to change it (a while ago, this is an old change). If I don't, then visualizations_controller.rb doesn't pass rubocop. For this file to pass rubocop with a cyclomatic complexity of 31, the displayVis function would have to completely rewritten/split up into several functions.